Dockerfile

```diff
@@ -3,10 +3,15 @@ FROM eclipse-temurin:21-jre-alpine
 RUN addgroup -S terraform-boot && adduser -S -G terraform-boot terraform-boot
 RUN apk update && \
     apk add --no-cache unzip wget
-ENV TERRAFORM_VERSION=1.4.4
-RUN wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip
-RUN unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip
-RUN mv terraform /usr/bin/terraform
+
+ENV TERRAFORM_INSTALL_PATH=/opt/terraform
+ENV DEFAULT_TERRAFORM_VERSION=1.6.0
+ENV TERRAFORM_VERSIONS=1.6.0,1.7.0,1.8.0,1.9.0
```

Comment:
we need the versions list also to be able to provided as an input to the github job. 

So the first requirement in the userstory would also mean that we update the [github action](https://github.com/eclipse-xpanse/terraform-boot/blob/main/.github/workflows/release.yml#L11) to let us provide the list of terraform versions just to provide some flexibility.